### DISCUSSION TITLE
Network monitoring

### DISCUSSION THREAD

**Discussion by thulsadume** [utmstack/UTMStack#614] [Q&A] [answered]
Does UTMStack do NDIS via a promiscuous interface?  If so, how do I configure it.  I have the interface connected and a tcpdump shows all the monitored traffic from the switch I connected it to.

**Comment by c3s4rfred** [comment, marked as answer]
Hi @thulsadume NIDS isn't available in the current version, it will be in one of the next releases.

Best regards
